preferences.py

```diff
@@ -156,7 +156,7 @@ class StableDiffusionPreferences(bpy.types.AddonPreferences):
     @staticmethod
     def register():
         if Pipeline.local_available():
-            set_model_list('installed_models', Generator.shared().hf_list_installed_models().result())
+            set_model_list('installed_models', Generator.shared().hf_list_installed_models(_block=True).result())
```

Comment:
Attempting to start Blender via the VS Code extension is currently freezing. Changing this will allow it to start for me but requires disabling and re-enabling the addon for anything that runs on the backend. Has a downside of the models not being available right away but, also doesn't slow down Blender since it doesn't have to wait on the backend to start.
```suggestion
            def on_done(future):
                set_model_list('installed_models', future.result())
            Generator.shared().hf_list_installed_models().add_done_callback(on_done)
```

Reply:
I think removing the delay from Blender's launch time is better anyways.